src/Augurk.Specifications/Configuration/RetentionPolicy.feature.cs

```diff
@@ -1,168 +0,0 @@
-﻿// ------------------------------------------------------------------------------
```

Comment:
These files shouldn't even have been in the Git repository in the first place, so I've removed them now (and added them to the .gitignore).